docs/src/examples/advection_supg.md

```diff
@@ -0,0 +1,39 @@
+## Streamline-Upwind/Petrov-Galerkin (SUPG) advection operator
+
+This is an example of the 1D scalar SUPG advection problem.
+
+### Problem formulation
+
+The advection problem is given by
+
+```math
+\dfrac{\partial u}{\partial t} + c \dfrac{\partial u}{\partial x} = 0
+```
+where ``u = u(x,t)`` and ``c`` is its associated phase speed.
+
+The SUPG advection weak form is given by
+
+```math
+\int_\Omega v \left(\dfrac{\partial u}{\partial t} \right) dV - \int_\Omega \dfrac{\partial v}{\partial x} \left(c u - c τ \left( \dfrac{\partial u}{\partial t} + c \dfrac{\partial u}{\partial x} \right) \right) dV = 0, \forall v \in V
+```
+for an appropriate test space ``V \subseteq H^1 \left( \Omega \right)`` on the domain.
+In this weak formulation, boundary terms have been omitted, as they are not present on the infinite grid for Local Fourier Analysis.
+Where ``τ`` is the scaling for SUPG, we note that ``τ = 0`` gives classical Galerkin formulation, ``τ = 0.5`` gives nodally exact solution to the advection-diffusion equation and ``τ = 1`` gives SUPG formulation.
```

Comment:
It's $\tau = h/2$ for nodally exact in steady pure advection. For advection-diffusion, the optimal/"exact" parameter depends on the cell Peclet number. I know the standard grid is $h=1$, but we should be clear about this. There is a more sophisticated expression for quadratic elements, that requires using different values at cell midpoints and edges (which is unnatural in a quadrature-based method and doesn't really help for multiple dimensions or non-uniform meshes, so isn't really used in practice).

Reply:
Alright thanks for clearing this, but this tau = h/2 will give nodally exact solution for linear elements